spec/views/pages/get-involved.html_spec.rb

```diff
@@ -1,23 +1,59 @@
 require "rails_helper"
 
 RSpec.describe("pages/get-involved", type: :view) do
+  include ExternalLinkHelper
+  include CmsLinkHelper
+
   before do
     render
   end
 
   it "has a title" do
-    expect(rendered).to have_css(".hero__text", text: "Help us inspire the computing experts of the future")
+    expect(rendered).to have_css(".hero__heading", text: "Play your part in inspiring the next generation")
+  end
+
+  it "has an intro section" do
+    expect(rendered).to have_css(".govuk-body-m", text: "We are dedicated to ensuring every child in England receives high-quality computing education")
+    expect(rendered).to have_css(".govuk-heading-s", text: "Demand is growing")
+  end
+
+  it "has an ambassador section" do
+    expect(rendered).to have_css(".govuk-heading-m", text: "Become a volunteer and then shout about it!")
+    expect(rendered).to have_link("STEM Ambassador", href: stem_ambassadors_url)
+  end
+
+  it "has a governor section" do
+    expect(rendered).to have_css(".govuk-heading-m", text: "Be an advocate for education")
+    expect(rendered).to have_link("Think about becoming a governor", href: governors_and_trustees_url)
+  end
+
+  it "has a video for both ambassador and governor sections" do
+    expect(rendered).to have_css(".banner-component__media-section", count: 2)
+  end
+
+  it "has a get in touch section" do
+    expect(rendered).to have_css(".govuk-heading-m", text: "Deliver social value through investing in improving young people’s life outcomes")
+    expect(rendered).to have_link("Get in touch", href: governors_and_trustees_url)
   end
 
   it "has a get involved section" do
```

Comment:
Perhaps rename to "making an impact today section"?

Reply:
The making an impact section is the one with the background below this I believe, so will probably leave as is for this